packages/itwinui-react/src/core/Tree/Tree.tsx

```diff
@@ -297,7 +297,7 @@ export const Tree = <T,>(props: TreeProps<T>) => {
                 'data-iui-virtualizer': 'item',
                 ref: virtualizer.measureElement,
                 style: {
-                  width: '100%',
+                  minInlineSize: '100%',
```

Comment:
Can this be set in the `useVirtualScroll` css or is it not common between the three components?

Reply:
I'm also wondering if this is a good default. Won't this always result in a horizontal overflow, even when the user wants truncation?

Reply:
I hadn't considered it common since the `TableRow` doesn't include `width` with it's virtualization styles specifically, but it does have `minWidth: 100%` on all rows. Moving the style to `useVirtualScroll` doesn't seem to create any visual changes, so it can be moved there. However, this style does always result in horizontal overflow, like you mentioned. Do you think that it might be better to ask users to add this style to their `TreeNodes` instead, so that truncation remains the default behavior?

Reply:
I think the `Tree` should behave consistently, whether virtualization is enabled or not. So yes, lets ask users to add custom CSS. In other words, just remove `width: 100%` from the inline style?

Ideally, the exact same CSS can be used to create horizontal scroll for both `enableVirtualization={false}` and `enableVirtualization={true}`.

Reply:
I've removed the inline width style in this [commit](https://github.com/iTwin/iTwinUI/pull/2170/commits/82304f067c2acd05f5b852bf8a1cbcd0dc030300). Also updated the stories to add inline styles to the `TreeNode` components, including the `Virtualized` story, since the nodes become much thinner than the `Tree` without `width: 100%`. Screenshot of `Virtualized` story without `width: 100%`:
![image](https://github.com/user-attachments/assets/33c2947d-8841-4b3d-95b7-9ebd74dbf3bf)

Also fixed a bug with the virtualized `Tree` where user defined inline styles weren't being merge with the styles being added to the virtual items, causing them to be lost.

Reply:
> the nodes become much thinner than the Tree without `width: 100%`

Hang on, this is a problem because it introduces another difference in behavior between virtualized and non-virtualized trees. The virtualized tree-nodes should fill the container by default, while allowing the user to easily override it.

So I'd suggest setting `--_iui-width: 100%` in the inline style instead of removing it altogether. The reason for using this variable over just `width: 100%` is that the user's CSS will be able to still easily set `width`.

After that, the "virtualized with horizontal scroll" story can set `width: fit-content` instead of `minInlineSize: 100%`.

Reply:
Setting `width: fit-content` on the virtualized horizontal scroll story causes the nodes to shrink to their content's width., which causes them to not fill the container. Screenshot: 
![image](https://github.com/user-attachments/assets/a433bc4e-0f06-4b3e-9a02-f3c9fe3a6487)

Is this okay? It feels like it should have `minWidth: 100%` as well, so that the minimum width of the nodes is the container's width.

Reply:
> It feels like it should have `minWidth: 100%` as well, so that the minimum width of the nodes is the container's width.

Yeah we can do both `width: fit-content` and `min-width: 100%`.

Reply:
Set the `--_iui-width: 100%` style for the virtual items and updated the react stories accordingly in this [commit](https://github.com/iTwin/iTwinUI/pull/2170/commits/07f0c639a0951e5a00f8b0f06728767e40aa1ca2).